### DISCUSSION TITLE
Map Statistics

### DISCUSSION THREAD

**Discussion by MiamiCenterPL** [diesel-modding/BeardLib-Editor#666] [Ideas]
Did you ever wonder how much of an Elements or Units your map has?
Or how much it took for Editor to load your map last time?
What I would like to see is a map statistics that shows these kind of things there, could be shown in console window for example.

**Comment by RexTheWho** [comment]
1. You can select all units/elements to get a number
2. There's a launch toggle called 'Check Load Time' to see how long it took to load

What I personally would want from debugging menus would be the debug info Overkill had where it tells you how many objects are visible, triangle count and proper performance stats.

**Reply by MiamiCenterPL** [reply]
It was only just a example of what it could show us.
But triangle count, performance stats and count of visible object is great idea!
